### DISCUSSION TITLE
Target class [App\Http\Controllers\1\ExampleController] does not exist.

### DISCUSSION THREAD

**Discussion by 250king** [laravel/framework#35590] [Q&A] [answered]
I want to create some controllers in which is not in App\Http\Controllers . But there are some mistakes here. I don't know should I do?

Screenshot:
![image](https://www.shirosaki-hana.fun/screenshot.png)

App\Http\Controllers\1\Index.php: 
```
<?php

namespace App\Http\Controllers;

class Index extends Controller
{
    /**
     * Create a new controller instance.
     *
     * @return object
     */
    public function send() {
        return response("v1");
    }
}

```

router/web.php
```
<?php

/** @var \Laravel\Lumen\Routing\Router $router */

$router->get('/', function () use ($router) {
    return $router->app->version();
});
$router->get("/1", "1\ExampleController@send");

```

**Comment by danilopolani** [comment, marked as answer]
Never used Lumen but your namespace is wrong - does not match what you're using in your router - and furthermore [you cannot use numbers as namespace](https://stackoverflow.com/questions/33875380/can-a-namespace-start-with-a-number-in-php).
